opencog/benchmark/AtomSpaceBenchmark.cc

```diff
@@ -68,14 +70,16 @@ AtomSpaceBenchmark::AtomSpaceBenchmark()
     doStats = false;
     testKind = BENCH_AS;
 
+    randomseed = (unsigned long) time(NULL);
+
+
     asp = NULL;
     atab = NULL;
 #if HAVE_CYTHON
     cogs = NULL;
     pymo = NULL;
 #endif
-
-    rng = new opencog::MT19937RandGen((unsigned long) time(NULL));
+    rng = NULL;
 }
```

Comment:
This was already here. I didn't introduce it. I made it possible to do something else.